php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S3984.html

```diff
@@ -0,0 +1,12 @@
+<p>Creating a new <code>Throwable</code> without actually throwing it is useless and is probably due to a mistake.</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+if (x &lt; 0)
+  new IllegalArgumentException("x must be nonnegative");
+</pre>
+<h2>Compliant Solution</h2>
+<pre>
+if (x &lt; 0)
+  throw new IllegalArgumentException("x must be nonnegative");
+</pre>
+
```

Comment:
I would use "Exception" instead of "Throwable" in the description (even if Throwable exists in PHP).
Variables are prefixed by '$', x => $x
Replace IllegalArgumentException by Exception.
Use simple quote to prevent $x to be replaced.
Ex:
```
if ($x < 0) {
  new  Exception('$x must be nonnegative'); // Noncompliant
}
```